doc/project/checklist.md

```diff
@@ -218,6 +218,11 @@ The security council decides whether an additional RTL review of the relevant co
 
 Any approved new features since D2 have been documented and reviewed with DV/SW/FPGA
 
+### DOC_PARAM_IP
+
+If not otherwise stated in the comportable IP specification, all instantiation parameter combinations are considered legal and do not impact D3/V3 status.
```

Comment:
This statement probably needs a bit more clarification. We do have parameterized blocks, but we only verify the parameter sets that go into the top level (i.e., ones that are going to be taped out). So all "parameter combinations may be considered legal", but only a subset of that is actually verified.

Reply:
yeah that seems like a good way to describe the reality. It partially makes me wonder if we need to actually expose this information in the `top_*` specification.  Because otherwise it seems to suggest that if the IP is fixed, and we can take on new top levels, the coverage would change and we would need to update the documentation to account for that.

Reply:
It almost sounds a bit like we need to split this into a design part and a DV part. 
Would it make sense to have such a checklist item on both the D3 and V3 checklists?
D3 would talk about what is considered legal, and V3 would talk about what has been verified?

Tim, to your point: re-parameterisation can definitely lead to coverage differences, to the point where even a legal parameter combo might need V3 re-certification at the block-level. Hence I have the impression that we probably need to keep this info somewhere in the block-level docs...

Reply:
It is already _sortof_ indicated in the V1 milestone checklist:
https://docs.opentitan.org/doc/project/checklist/#tb_gen_automated

It does make sense to formalize a flow for SoCs to provide the list of configurations so that they all can be verified at the IP level.